src/Twig/LinkFactory.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace PhpDocumentGenerator\Twig;
+
+use PhpDocumentGenerator\ReflectionNamedType as PDGReflectionNamedType;
+use PHPStan\PhpDocParser\Ast\Type\TypeNode;
+use PHPStan\PhpDocParser\Ast\Type\UnionTypeNode;
+use ReflectionClass;
+use ReflectionNamedType;
+use ReflectionType;
+use Symfony\Component\Filesystem\Path;
+
+final class LinkFactory
```

Comment:
A factory is a design pattern for creating objects, but no object is created in this factory. A trait or a helper would be better